lib/std/Build/CompileStep.zig

```diff
@@ -144,6 +144,8 @@ link_function_sections: bool = false,
 /// exported symbols.
 link_gc_sections: ?bool = null,
 
+linker_dynamicbase: ?bool = null,
```

Comment:
I think this is a Windows specific flag so as such we should have a doc-style comment that makes this clear.

Reply:
Agreed, added.